python/cac_tripplanner/destinations/migrations/0007_destination_website_url.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('destinations', '0006_add_city_state_defaults'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='destination',
+            name='website_url',
+            field=models.URLField(default='http://github.com/azavea/cac-tripplanner'),
```

Comment:
Might be better to show an error to the user if they haven't entered a link, or make the field not be required; we probably don't want this link to make it into production use.